lib/matplotlib/_docstring.py

```diff
@@ -3,6 +3,31 @@
 from . import _api
 
 
+def kwarg_doc(text):
+    """
+    Decorator for defining the documentation used when artist properties are
+    passed through as keyword arguments.
+
+    See e.g. the ``**kwargs`` section in `.Axes.text`.
+
+    The given text is stored in a privat variable ``_kwarg_doc`` on the method.
```

Comment:
```suggestion
    The given text is stored in a private attribute ``_kwarg_doc`` on the method.
```

Reply:
Should this also explicitly say it is for use on the Artist property *setter*, not just any random method.